docs/tutorial.rst

```diff
@@ -407,6 +407,225 @@ filter as such in the view:
 
             return qs
 
+Renaming forwarded values
+-------------------------
+- Example source code: `test_project/rename_forward
+  <https://github.com/yourlabs/django-autocomplete-light/tree/master/test_project/rename_worward>`_.
+- Live demo: `Admin / Rename Forward/ Add
+  <http://dal-yourlabs.rhcloud.com/admin/rename_forward/testmodel/add/>`_.
+
+Let's assume that you have the following form using linked autocomplete fields:
+
+.. code-block:: python
+
+    class ShippingForm(forms.Form):
+        src_continent = forms.ModelChoiceField(
+            queryset=Continent.objects.all(),
+            widget=autocomplete.ModelSelect2(url='continent-autocomplete'))
+        src_country = forms.ModelChoiceField(
+            queryset=Country.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='country-autocomplete',
+                forward=('src_continent',)))
+        src_city = forms.ModelChoiceField(
+            queryset=City.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='city-autocomplete',
+                forward=('src_country',)))
+        # More source fields...
+
+        dst_continent = forms.ModelChoiceField(
+            queryset=Continent.objects.all(),
+            widget=autocomplete.ModelSelect2(url='continent-autocomplete'))
+        dst_country = forms.ModelChoiceField(
+            queryset=Country.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='country-autocomplete',
+                forward=('dst_continent',)))
+        dst_city = forms.ModelChoiceField(
+            queryset=City.objects.all(),
+            widget=autocomplete.ModelSelect2(
+                url='city-autocomplete',
+                forward=('dst_country',)))
+        # More destination fields...
```

Comment:
Would it work to show just one field and rename it ? I don't think we have
to prove the use case in the tutorial, just show the feature in the most
minimalist way.

On Sep 11, 2016 1:00 PM, "Kirill Gagarski" <notifications@github.com> wrote:

> In docs/tutorial.rst
> <https://github.com/yourlabs/django-autocomplete-light/pull/733#discussion_r78292526>
> :
>
> > +        # More source fields...
> > +
> > +        dst_continent = forms.ModelChoiceField(
> > +            queryset=Continent.objects.all(),
> > +            widget=autocomplete.ModelSelect2(url='continent-autocomplete'))
> > +        dst_country = forms.ModelChoiceField(
> > +            queryset=Country.objects.all(),
> > +            widget=autocomplete.ModelSelect2(
> > +                url='country-autocomplete',
> > +                forward=('dst_continent',)))
> > +        dst_city = forms.ModelChoiceField(
> > +            queryset=City.objects.all(),
> > +            widget=autocomplete.ModelSelect2(
> > +                url='city-autocomplete',
> > +                forward=('dst_country',)))
> > +        # More destination fields...
>
> Like limiting fields to country/city? Or to only one
> continent/country/city field set? Or both?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/yourlabs/django-autocomplete-light/pull/733/files/f68ca745ad7c2417405eba85b06bf1453eb139bf#r78292526>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFxrAQJYztQXnWVhhfptMzTG_hdNrROks5qo988gaJpZM4JzLkD>
> .
>